core/models.py

```diff
@@ -29,12 +29,30 @@ class Modules(models.Model):
     create_by = models.ForeignKey(User, on_delete=models.CASCADE, related_name='md_created_by', null=True)
 
     class Meta:
-        verbose_name_plural = 'User Modules'
-        db_table = 'jinoe_user_modules'
```

Comment:
Good Job  so far 👍  , but we recommend to   make a fix branch when ever there is anything to fix.  Why? To avoid  mixed commits